ai/states/game_state.py

```diff
@@ -111,6 +113,10 @@ def enemy_team(self) -> Team:
     def ball(self) -> Ball:
         return self._field.ball
 
+    @ball.setter
+    def ball(self, ball: Ball) -> Ball:
+        self._field = Field([ball])
```

Comment:
we need to reset the field to change the ball position? seems odd

Reply:
I was lazy, I did not wanted a setter just for perfect_sim

Reply:
It should at least be comment about :p We shouldn't use this method in the main code haha